ui/client/components/graph/node-modal/editors/expression/Duration/DurationEditor.tsx

```diff
@@ -50,29 +46,14 @@ export default function DurationEditor(props: Props) {
 
   function decode(expression: string): Duration {
     const regexExec = SPEL_DURATION_DECODE_REGEX.exec(expression)
-    const duration = regexExec === null ? UNDEFINED_DURATION : moment.duration(regexExec[1])
+    const duration = regexExec === null ? NULL_DURATION : moment.duration(regexExec[1])
```

Comment:
This should check `regexExec == null`  regexExec == null or regexExec == undefined.

Reply:
Done